tasks/eva_2380/find_FP_variants.py

```diff
@@ -0,0 +1,161 @@
+import os
+import sys
+from argparse import ArgumentParser
+from collections import defaultdict
+
+import pysam
+
+
+def fetch_bases(fasta, contig, start, length):
+    """
+    Returns a subsection from a specified FASTA contig. The start coordinate is 1-based.
+    """
+    zero_base_start = start - 1
+    end = zero_base_start + length
+    new_ref = fasta.fetch(reference=contig, start=zero_base_start, end=end)
+    return new_ref
+
+
+def assessment_result_per_variants_from_vcf(vcf_file):
```

Comment:
might consider renaming this method to highlight the fact that we're focusing on false positives?